Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+
+node {
+	def VERSION = sh(returnStdout: true, script: "git rev-parse HEAD").trim()
+	def CACHE_DIR = sh(returnStdout: true, script: "mktemp go-ipfs.cache.XXXXXXXX").trim()
```

Comment:
Why is the cache separated from docker? With a correct Dockerfile, packages would already be cached from previous layers created.